src/jquery.cycle2.carousel.js

```diff
@@ -156,7 +156,6 @@ $.fn.cycle.transitions.carousel = {
         function onResize() {
             opts._carouselWrap.stop( false, true );
             var slideWidth = opts.container.width() / opts.carouselVisible;
-            slideWidth = Math.ceil( slideWidth - adjustment );
```

Comment:
I disagree with removing the rounding. In the stuff I am doing, removing this rounding, causes images to shift pixels. While it might work for most use cases there are some instances that still cause problems.